### ISSUE TITLE
Fireworks Effect

### ISSUE THREAD

**Issue by gleeds** [ahodges9/LedFx#45]
I saw this neat fireworks effect on reddit (includes sample code):
https://www.reddit.com/r/arduino/comments/c3sd46/i_made_this_fireworks_effect_for_my_led_strips/

Something like this could look really cool synchronized to music.

**Comment by ahodges9** [comment]
@not-matt, could we do something similar with a new effectlet? None of the quite give this effect, but they are close. 

@gleeds, check out all "rain" effect on the latest release and the various settings it has.

**Comment by not-matt** [comment]
Yup that's possible, I can write a new effectlet to look like that
